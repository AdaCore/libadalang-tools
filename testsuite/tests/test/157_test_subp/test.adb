package body Test is

   procedure Test (X : My_Int; Y : Integer; Z : Foo.Test) is null;

end Test;
