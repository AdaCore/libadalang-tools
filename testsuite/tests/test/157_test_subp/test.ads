with Foo; use Foo;

package Test is

   type My_Int is new Integer;

   procedure Test (X : My_Int; Y : Integer; Z : Foo.Test);

end Test;
