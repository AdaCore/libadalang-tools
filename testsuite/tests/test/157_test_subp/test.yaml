description: Check that the support library and gnattest harness are legal when
  either the subprogram under test is called "test" or the unit in which it is
  defined is called test.

driver: gnattest_driver

control:
    - [XFAIL, 'x86', 'Marshalling not working for 32bits (UB03-008)']
