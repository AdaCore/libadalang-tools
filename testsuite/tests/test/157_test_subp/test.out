test.ads:7:4: info: corresponding test PASSED
test.ads:7:4: info: corresponding test PASSED
test.ads:7:4: info: corresponding test PASSED
test.ads:7:4: info: corresponding test PASSED
test.ads:7:4: info: corresponding test PASSED
test.ads:7:4: error: corresponding test FAILED: Test not implemented. (test-test_data-tests.adb:45)
6 tests run: 5 passed; 1 failed; 0 crashed.
